_integration-schemas/ms-teams/v1/conversation_threads.md

```diff
@@ -0,0 +1,55 @@
+---
+tap: "ms-teams"
+version: "1"
+key: ""
+
+name: "conversation_threads"
+doc-link: "https://docs.microsoft.com/en-us/graph/api/conversation-list-threads?view=graph-rest-beta&tabs=http"
+singer-schema: "https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/schemas/conversation_threads.json"
+description: |
+  The `{{ table.name }}` table contains information about threads in a group conversation in your Microsoft account.
+
+replication-method: "Key-based Incremental"
+
+api-method:
+    name: "List threads"
+    doc-link: "https://docs.microsoft.com/en-us/graph/api/conversation-list-threads?view=graph-rest-beta&tabs=http"
```

Comment:
```suggestion
    doc-link: "https://docs.microsoft.com/en-us/graph/api/group-list-threads?view=graph-rest-1.0&tabs=http"
```

[This table actually uses the V1 API ](https://github.com/singer-io/tap-ms-teams/blob/master/tap_ms_teams/streams.py#L443)